Library/Homebrew/hardware.rb

```diff
@@ -72,7 +72,7 @@ def family
 
       def cores
         return @cores if @cores
-        @cores = Utils.popen_read("getconf", "_NPROCESSORS_ONLN").chomp.to_i
+        @cores = Utils.popen_read("sysctl", "-n", "hw.physicalcpu").chomp.to_i
```

Comment:
You'd probably also want to take advantage of hyperthreading -- `hw.logicalcpu` might be better?